gitpod-io/gitpod
# [gitpod-cli] clean up

## Description


This PR introduces SupervisorClient to cache grpc connection and clients. It is in a prepartion to support inner container while running gp-run to provide access to all tasks.

## Related Issue(s)

Fixes #

## How to test


- Open https://github.com/akosyakov/parcel-demo
- You should see preview and file opened.
- Test following gp commands: info, top, open, preview, tasks and ports management.

## Release Notes

```release-note
NONE
```

## Documentation


## Werft options:

      If enabled this will build `install/preview`
      Valid options are `all`, `workspace`, `webapp`, `ide`, `jetbrains`, `vscode`, `ssh`
